html-forms/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Form</title>
+</head>
+<body>
+  <form action="" method="get">
+    <fieldset>
+      <div>
+        <label for="email">Email</label>
+      </div>
+      <input type="text" name="email" id="email">
+      <div>
+        <label for="dropdown">What is coding cool?</label>
+      </div>
+      <select id="dropdown" name="dropdown">
+        <option value="option">Select an option</option>
+        <option value="interesting">Because it's interesting</option>
+        <option value="fun">Because it's fun</option>
+        <option value="powerful">Because it's powerful</option>
+      </select>
+        <div>Favorite Language (choose one)</div>
+        <label><input type="radio" name="language" value="html">HTML</label>
+        <label><input type="radio" name="language" value="css">CSS</label>
+        <label><input type="radio" name="language" value="js">Javascript</label>
+      <div>
+        <label for="signup">Would you like to sign up for more information?</label>
+      </div>
+        <input type="checkbox" name="signup" value="signup" id="signup">Sign me up!
+      <div>
+        <label for="comments">Comments</label>
+      </div>
+      <textarea name="comments"></textarea>
```

Comment:
make sure to add the id attribute here to match the label above

Reply:
Thanks for the feedback!